.github/CODEOWNERS

```diff
@@ -78,12 +78,12 @@
 /apps/download-service/                                                                       @island-is/sendiradid
 /libs/clients/documents/                                                                      @island-is/sendiradid
 
-/apps/services/search-indexer/                                                                @island-is/stefna @island-is/kosmos-kaos @island-is/aranja
```

Comment:
Is `@island-is/kosmos-kaos` removed on purpose for the following paths?

Reply:
Please re-review.